tests/fixtures/requests/response-parsing.yml

```diff
@@ -0,0 +1,238 @@
+                          </productRequest>
+                          <productRequest formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="BG"/>
+                          </productRequest>
+                          <productRequest requestedDeadline="2019-04-15T12:00:00+02:00" formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="CS"/>
+                          </productRequest>
+                          <productRequest requestedDeadline="2019-11-15T12:00:00+01:00" formatCode="NA" statusCode="ENV" trackChanges="false">
+                              <language code="DE"/>
+                          </productRequest>
+                      </productRequests>
+                      <auxiliaryDocuments>
+                          <auxiliaryDocument format="TXT" type="REF" name="test1.txt" language="ML"/>
+                          <auxiliaryDocument format="TXT" type="REF" name="test1.txt" language="RO"/>
+                      </auxiliaryDocuments>
+                  </return>
+              </ns0:createRequestsResponse>
+          </S:Body>
+      </S:Envelope>
+    request:
+      linguisticRequest:
+        - contacts:
+            contact:
+              - UserId: '1'
+                RoleCode: AUTHOR
+              - UserId: '2'
+                RoleCode: EDITOR
+          generalInfo:
+            Title: Test
+            InternalReference: '1'
+            InternalTechnicalId: '1'
+            RequestedDeadline:
+              time: '2020-02-02'
+              timezone: null
+            Sensitive: false
+            DocumentToBeAdopted: true
+            DecideReference: decideReference
+            SentViaRUE: true
+            DestinationCode: PUBLIC
+            SlaAnnex: ''
+            SlaCommitment: ''
+            Comment: ''
+            OnBehalfOf: ''
+            AccessibleTo: ''
+  expectations:
+    request: |
+      <?xml version="1.0" encoding="UTF-8"?>
+      <SOAP-ENV:Envelope xmlns:SOAP-ENV="http://schemas.xmlsoap.org/soap/envelope/" xmlns:ns1="http://eu.europa.ec.dgt.epoetry">
+      <SOAP-ENV:Body>
+      <ns1:createRequests>
+      <linguisticRequest><generalInfo><title>Test</title><internalReference>1</internalReference><internalTechnicalId>1</internalTechnicalId><requestedDeadline>2020-02-02T00:00:00+00:00</requestedDeadline><sensitive>false</sensitive><documentToBeAdopted>true</documentToBeAdopted><decideReference>decideReference</decideReference><sentViaRUE>true</sentViaRUE><destinationCode>PUBLIC</destinationCode><slaAnnex></slaAnnex><slaCommitment></slaCommitment><comment></comment><onBehalfOf></onBehalfOf><accessibleTo></accessibleTo></generalInfo>
+      <contacts><contact userId="1" roleCode="AUTHOR"/><contact userId="2" roleCode="EDITOR"/></contacts>
+      </linguisticRequest>
+      </ns1:createRequests>
+      </SOAP-ENV:Body>
+      </SOAP-ENV:Envelope>
+    assertions:
```

Comment:
We can convert these assertions in a simple list of expressions, such as:

```yml
    assertions:
      - 'count(response.getReturn()) === 2'
      - 'response.getReturn()[0].getStatusCode() === ENV'
```
This will make tests easier to write and more extensible. Note that `count()` should be registered as indicated in my previous comment.

Reply:
Fixed.